### ISSUE TITLE
POST then GET - Body not cleared (FF)

### ISSUE THREAD

**Issue by moderndeveloperllc** [esphen/RESTED#105]
Steps to reproduce on Firefox 54:

1. Create a Collection that had a POST with a body (not form attributes).
2. Create a Collection that has a GET call.
3. Run the POST call.
4. Run the GET call.

```
An error occured while fetching the resource: TypeError: HEAD or GET Request cannot have a body.
```

This error does not occur with Chrome/Opera. GET calls work fine if you don't first run the POST that had a body. If I change the GET to a POST for the second call, I can see that the body was not cleared when changing between collections and calls.

**Comment by esphen** [comment]
Yep, this was fixed in `2.0.4`, but Mozilla is taking their sweet time reviewing it. 12 days and counting. They're probably all on vacation :)

Thanks for the report

**Comment by moderndeveloperllc** [comment]
Oops. Didn't read the release notes well enough!
